horizons/gui/widgets/minimap.py

```diff
@@ -279,18 +281,64 @@ def update_cam(self):
 			return # don't draw while loading
 		self.minimap_image.set_drawing_enabled()
 		self.minimap_image.rendertarget.removeAll(self._get_render_name("cam"))
+
+		def drawLine(p1, p2):
+			self.minimap_image.rendertarget.addLine(self._get_render_name("cam"),
+			                                        p1,
+			                                        p2,
+			                                        *self.COLORS["cam"])
+
 		# draw rect for current screen
 		displayed_area = self.view.get_displayed_area()
-		minimap_corners_as_point = []
-		for (x, y) in displayed_area:
-			coords = self.transform.world_to_minimap((x, y))
-			minimap_corners_as_point.append(fife.Point(coords[0], coords[1]))
 
-		for i in range(0, 4):
-			self.minimap_image.rendertarget.addLine(self._get_render_name("cam"),
-			                                        minimap_corners_as_point[i],
-			                                        minimap_corners_as_point[(i + 1) % 4],
-			                                                         *self.COLORS["cam"])
+		xmin, ymin = self.transform.world_to_minimap(displayed_area[0])
+		xmax, ymax = self.transform.world_to_minimap(displayed_area[2])
+
+		# draw horizontal lines
+		for y in [ymin, ymax]:
```

Comment:
Tuples are lighter than lists so it's slightly better to use a tuple here:
```python
for y in (ymin, ymax):
```

Reply:
I thought I deleted this code in a later commit?

Reply:
My bad, yes, you did.